### DISCUSSION TITLE
Hakui Koyori真可愛

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2144] [General]
Hakui Koyori（博衣小夜璃）是HoloX組織的科學家（頭腦）。我認為我們應該和博衣小夜璃一樣進行科學研究，藉此實現統治世界的目標！
我個人建議大家可以可以嘗試和博衣小夜璃學習一些科學知識，也可以學習一些直播的技巧，增進社交能力！
更具體來說，我建議大家可以前往博衣小夜璃的youtube頻道https://www.youtube.com/@HakuiKoyori 訂閱並準時觀看直播。
另一個建議是，我們可以加入博衣小夜璃的頻道會員，這樣便可以觀看會員限定影片，獲取更多知識！

**Comment by gigitilto** [comment]
請問這個頻道可以幫助我學習到哪方面的知識呢？

**Reply by LeoTsai999** [reply]
我認為這個頻道對於研究奇怪的藥物非常有幫助！
我建議可以在學習配置奇怪的藥物後，思考如合用它來統治世界
